src/openttd.cpp

```diff
@@ -650,7 +654,7 @@ int openttd_main(int argc, char *argv[])
 		 *
 		 * The next two functions are needed to list the graphics sets. We can't do them earlier
 		 * because then we cannot show it on the debug console as that hasn't been configured yet. */
-		DeterminePaths(argv[0]);
+		DeterminePaths(argv[0], false);
```

Comment:
Shouldn't this be `only_local_path`? Otherwise `openttd -X -h` lists base sets that `openttd -X -I/S/M ...` cannot find